confluent_kafka/avro/__init__.py

```diff
@@ -114,7 +116,7 @@ def poll(self, timeout=None):
             if message.value() is not None:
                 decoded_value = self._serializer.decode_message(message.value())
                 message.set_value(decoded_value)
-            if message.key() is not None:
+            if self._enable_key_decoding and message.key() is not None:
```

Comment:
Looks like when `_enable_key_decoding` is `False` the resulting message will not have the key set. Is that right?